app/components/post_list/post_list.js

```diff
@@ -281,7 +281,7 @@ export default class PostList extends PureComponent {
 
     scrollToBottom = () => {
         setTimeout(() => {
-            this.flatListRef.current.scrollToOffset({offset: 0, animated: true});
+            this.flatListRef.current?.scrollToOffset({offset: 0, animated: true});
```

Comment:
isn't this supposed to be 
```suggestion
            this.flatListRef?.current.scrollToOffset({offset: 0, animated: true});
```

Reply:
@migbot also the way you have it is failing tests
```
/home/ubuntu/jenkins/workspace/mm/mattermost-mobile-pr/app/components/post_list/post_list.js
  284:13  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions

✖ 1 problem (1 error, 0 warnings)
```